scripts/just.config.ts

```diff
@@ -56,8 +55,7 @@ export function preset() {
   task('sass', sass());
   task('ts:postprocess', postprocessTask());
   task('postprocess:amd', postprocessAmdTask);
-  task('postprocess:commonjs', postprocessCommonjsTask);
-  task('ts:commonjs', series(ts.commonjs, 'postprocess:commonjs'));
```

Comment:
Mentioned in the comments ,but this is the postprocess task which is mapping the `/lib/` imports in the cjs files to `/lib-commonjs/`. But that's no longer needed with exports maps that will resolve a require for `/lib/Button` to a cjs implementation. And in fact, it is breaking scenarios where the file it transforms points to a file which isn't being exposed by the exports map. E.g. if you use charting today and import it from a cjs jest test, it will load the cjs charting binaries, which will import `lib-commonjs/Button` from fluent, which is not an entry currently in the exports map.

So we don't need this transform anymore.

Reply:
gotcha, thanks for more clarification!

can you please remove the `postprocessCommonjsTask` as well as it wont be needed anymore ? ty!

Reply:
done!